go/Dockerfile

```diff
@@ -9,7 +9,8 @@ apt install -y --no-install-recommends \
     build-essential=12.9ubuntu3 \
     ca-certificates \
     g++-riscv64-linux-gnu=4:11.2.0--1ubuntu1 \
-    wget
+    wget \
+    curl
```

Comment:
Leak? Or only wrong sorting?

Reply:
this commit was just to trigger the CI

not related to the PR itself